oadueslookup.php

```diff
@@ -171,6 +170,27 @@ function oadueslookup_update_db_check()
     add_option('oadueslookup_last_import', '1900-01-01');
     add_option('oadueslookup_last_update', '1900-01-01');
     add_option('oadueslookup_max_dues_year', '2016');
+
+}
+
+function oadueslookup_update_shortcodes()
+{
+    # In version 2.1, we replaced the URL trap with a shortcode.
+    # This code converts from the old way to the new way.
+    $lookup_slug = get_option('oadueslookup_slug', 'it was not set');
+    if (!($lookup_slug == 'it was not set')) {
```

Comment:
@justdave consider using `===` instead of `==`

Reply:
I concur, done.